app/main.py

```diff
@@ -1,6 +1,30 @@
-from typing import Callable
+from typing import Callable, Any
+from functools import wraps
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    @wraps(func)
+    def wrapper(*args, **kwargs) -> Any:
+        key = args + tuple(sorted(kwargs.items()))
+        if key in cache_dict:
+            print("Getting from cache")
+            return cache_dict[key]
+        else:
```

Comment:
There is no need for `else` since the code will not execute after the `return` statement anyway. Please, remove the redundant `else`